### PR TITLE
PMM-7467 Fix bug with angle brackets escaping in rule expressions

### PR DESCRIPTION
**Pull request by artemgavrilov**
[PMM-7467](https://jira.percona.com/browse/PMM-7467)

https://github.com/Percona-Lab/pmm-api-tests/pull/182

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1464

---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: services/management/ia/templates_service.go [hunk 1]
```diff
@@ -19,12 +19,12 @@ package ia
 import (
 	"bytes"
 	"context"
-	"html/template"
 	"io/ioutil"
 	"path/filepath"
 	"sort"
 	"strings"
 	"sync"
+	"text/template"
```
Comment (AlekSi) [inline services/management/ia/templates_service.go:27]: Epic 😆 Please check other places too

Comment (artemgavrilov) [reply services/management/ia/templates_service.go:27]: Already checked, seems that it the only place where we execute expression)

**Review by Dasio — APPROVED** [review]
What about already existing templates. Can we fix it somehow?

Or we don't care about it sine it's beta.

**Comment by artemgavrilov** [comment]
> What about already existing templates. Can we fix it somehow?
> 
> Or we don't care about it sine it's beta.

That bug affects rules, not templates. If user created a rule that has `<` it expression, then he cannot use it anyway. So yes, he should just recreate the rule, because as you said it's beta and the fix will be too complex for that.

**Review by BupycHuk — APPROVED** [review]

**Review by oter — APPROVED** [review]

**Pull request merged**